index.js

```diff
@@ -20,6 +20,36 @@ StylusFilter.prototype.targetExtension = 'css';
 
 StylusFilter.prototype.processString = function (str) {
 	var opts = this.options;
+
+	var importRegex = new RegExp(/@import ['"].*['"]/);
```

Comment:
Should this be it's own module, just like we have https://github.com/kevva/import-regex?